src/axes/categoryAxis.ts

```diff
@@ -1,15 +1,7 @@
 namespace Plottable.Axes {
-  export interface DownsampleInfo {
-    domain: string[];
-    stepWidth: number;
-  }
+  export type ITickMarkPosition = "center" | "after";
```

Comment:
Why would people use "after"? I'd prefer a "between" (that puts ticks between the labels)